src/Temporalio/Worker/Tuning/SlotPermit.cs

```diff
@@ -0,0 +1,33 @@
+namespace Temporalio.Worker.Tuning
+{
+    /// <summary>
+    /// A permit to use a slot for a workflow/activity/local activity task.
+    /// </summary>
+    /// <remarks>
+    /// WARNING: Custom slot suppliers are currently experimental.
+    /// </remarks>
+    public class SlotPermit
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SlotPermit"/> class with no associated data.
+        /// </summary>
+        public SlotPermit()
+        {
+            UserData = null;
+        }
```

Comment:
```suggestion
        public SlotPermit() => UserData = null;
```

Just an FYI, lots of these single-line methods can be shortened. But this is pedantic/unimportant.